### PR TITLE
Update docs for Cloud Run with new beta deploy command

### PR DESCRIPTION
**Pull request by subfuzion**

### CODE REVIEW

File: docs/quick-starts/03-quick-start-cloud-run.md [hunk 1]
```diff
@@ -27,15 +30,72 @@ Make sure that billing is enabled for your project ([see how]).
...
+```
+
 ## Get a copy of the `hello` example
 
 Clone this repo or download a [zip] archive and extract the contents. Change
 directory to `examples/hello`.
 
+## Build and deploy with a single command
+
+The next two sections cover two commands that can be replaced with a single
```
Comment (kevmoo) [inline docs/quick-starts/03-quick-start-cloud-run.md:54]: Maybe something like

```suggestion
The next two sections cover one command (now in beta) that can replace two commands.
```

**Review by kevmoo — DISMISSED** [review]

File: docs/quick-starts/03-quick-start-cloud-run.md (hunk 1 above)
Comment (subfuzion) [reply docs/quick-starts/03-quick-start-cloud-run.md:54]: Hmm that's not better either, but agree this should and can be reworded to read more smoothly. Gimme a moment...

Comment (subfuzion) [reply docs/quick-starts/03-quick-start-cloud-run.md]: How's this?

**Review by kevmoo — APPROVED** [review]
:shipit:

**Pull request merged**